media/static-bundles.json

```diff
@@ -479,22 +479,17 @@
     },
     {
       "files": [
-        "css/firefox/privacy/common.scss"
-      ],
-      "name": "firefox-privacy-common"
-    },
-    {
-      "files": [
-        "css/firefox/privacy/promise.scss"
+        "css/firefox/whatsnew/whatsnew-127-eu.scss"
```

Comment:
**Issue:** Not sure why the whatsnew-127 files are showing up as being added here. Rebase weirdness?

Reply:
It was some glitch with rebase. I have rebase again and the git history is as expected now